test/unit/preloaded-vl-proza-message.test.html

```diff
@@ -82,6 +85,24 @@
                 assert(console.error.called);
             });
         });
+
+        test('een bericht wordt individueel opgehaald indien het niet in de preload cache gevonden kan worden', () => {
```

Comment:
Dit is code voor wanneer de back-end niet correct al de messages doorgeeft?

Reply:
Dat is ook een effect van de code, ja. Maar het probleem waar we tegenaan liepen en wat dit eigenlijk echt oplost was dat we in de backend van een applicatie niet (gemakkelijk kunnen) weten welke codes er in de front-end gebruikt worden. We weten dan niet welke berichten we moeten teruggeven wanneer iets gepreload wordt, in de dummy proza controller van de applicatie.
In plaats van manueel een lijst van codes bij te houden in de backend of in plaats van een tool te schrijven die dat uit de front-end gaat parsen, lossen we het hier op door indien de preload faalt of indien het bericht niet gevonden kan worden in de cache, toch het individuele bericht nog eens op te vragen.
In normale cases gaat dit dus niet gebeuren, maar het voegt gewoon wat extra robuustheid toe.